test/mpi/test_mpi.cc

```diff
@@ -288,6 +288,36 @@ void UccTestMpi::set_ops(std::vector<ucc_reduction_op_t> &_ops)
     ops = _ops;
 }
 
+int ucc_coll_reduce_supported(ucc_reduction_op_t op, ucc_datatype_t dt)
+{
+    switch (dt) {
+    case UCC_DT_INT8:
+    case UCC_DT_INT16:
+    case UCC_DT_INT32:
+    case UCC_DT_INT64:
+    case UCC_DT_INT128:
+    case UCC_DT_UINT8:
+    case UCC_DT_UINT16:
+    case UCC_DT_UINT32:
+    case UCC_DT_UINT64:
+    case UCC_DT_UINT128:
+        return (op != UCC_OP_AVG);
+    case UCC_DT_FLOAT16:
+    case UCC_DT_FLOAT32:
+    case UCC_DT_FLOAT64:
+    case UCC_DT_BFLOAT16:
+    case UCC_DT_FLOAT128:
+        return (op == UCC_OP_SUM || op == UCC_OP_PROD || op == UCC_OP_MAX ||
+                op == UCC_OP_MIN);
```

Comment:
does that mean that `UCC_OP_AVG` for float types is disabled?

Reply:
you are right, fixed, thanks